hexrdgui/mask_compatability.py

```diff
@@ -0,0 +1,31 @@
+from hexrd.utils.compatibility import h5py_read_string
+
+
+def load_old_mask_file(self, h5py_group):
```

Comment:
This can be changed to `load_masks_v1_to_v2` instead since it is converting masks from version 1 to version 2. Also, `self` shouldn't be an argument.

Reply:
Done